requirements.in

```diff
@@ -53,6 +53,7 @@ markupsafe==2.1.2
 mccabe==0.7.0
     # via flake8
     # via -r -
+mozilla-nimbus-schemas==2023.6.6
```

Comment:
does this need to be added as a requirement in `setup.py`?

Reply:
I ask because of CI failures on https://github.com/mozilla/bigquery-etl/pull/4007

Reply:
yes